katago_server/distributed_efforts/models/configuration/dynamic_distributed_task.py

```diff
@@ -15,6 +15,9 @@ class Meta:
     katago_config = TextField(
         _("katago config"), help_text=_("The configuration file for katago to be given to client for selfplay"), default="FILL ME"
     )
+    # QUESTION (lightvector): I assume "probability_predefined_training_game" is supposed to be the games that start from specified SGF positions?
```

Comment:
Okay, I thought it was weird that we would have ranking games start in predefined positions. But I guess in theory it could be used to force starting with different opening moves or something.

Reply:
Exactly